scheduler/test/cook/test/mesos/scheduler.clj

```diff
@@ -899,7 +899,16 @@
                             :where
                             [?i :instance/reason ?r]
                             [?r :reason/name ?reason-name]]
-                          (db conn) instance-id))))))
+                          (db conn) instance-id))))
+        (let [get-end-time (fn [] (ffirst (q '[:find ?end-time
+                                               :in $ ?i
+                                               :where
+                                               [?i :instance/end-time ?end-time]]
+                                             (db conn) instance-id)))
+              original-end-time (get-end-time)]
+          (async/<!! (sched/handle-status-update conn driver fenzo
+                                                 (make-dummy-status-update task-id :reason-gc-error :task-killed)))
+          (is (= original-end-time (get-end-time))))))
```

Comment:
Nitpick: It is possible that this test executes so fast that the two `now` return the same value? We should introduce a delay or `with-redef` `now` to ensure that the status message processing is happening at a different time.